code/__DEFINES/flags.dm

```diff
@@ -27,23 +27,63 @@ GLOBAL_LIST_INIT(bitflags, list(1, 2, 4, 8, 16, 32, 64, 128, 256, 512, 1024, 204
 #define DF_ISPROCESSING (1<<2)
 
 //! ## FLAGS BITMASK
-#define CONDUCT_1					(1<<5)		//!  conducts electricity (iron etc.)
-#define NODECONSTRUCT_1				(1<<7)		//!  For machines and structures that should not break into parts, eg, holodeck stuff
-#define OVERLAY_QUEUED_1			(1<<8)		//!  atom queued to SSoverlay
-#define ON_BORDER_1					(1<<9)		//!  item has priority to check when entering or leaving
-#define PREVENT_CLICK_UNDER_1		(1<<11)		//! Prevent clicking things below it on the same turf eg. doors/ fulltile windows
-#define HOLOGRAM_1					(1<<12)
-#define TESLA_IGNORE_1				(1<<13) 	//! TESLA_IGNORE grants immunity from being targeted by tesla-style electricity
-#define INITIALIZED_1				(1<<14)  	//! Whether /atom/Initialize(mapload) has already run for the object
-#define ADMIN_SPAWNED_1				(1<<15) 		//! was this spawned by an admin? used for stat tracking stuff.
-#define PREVENT_CONTENTS_EXPLOSION_1 (1<<16)
-#define UNPAINTABLE_1 				(1<<17)
-#define HTML_USE_INITAL_ICON_1		(1<<18) 			//! Should we use the initial icon for display? Mostly used by overlay only objects
+
+/// conducts electricity (iron etc.)
+#define CONDUCT_1 (1<<1)
```

Comment:
This just looked. so. bad.

i cleaned it up and actually ordered the bits in a *legible* and sane manner.

Last person who touched it left these... gaps? between the numbering. Im guessing thats because they were trying to account for TG definitions. 
I just brought those definitions over and commented them, so the spirit is kept, but its also not confusing as hell as to why the numbering is all over.